libass/ass_render.c

```diff
@@ -1300,9 +1300,11 @@ static void calc_transform_matrix(ASS_Renderer *render_priv,
  */
 static void
 get_bitmap_glyph(ASS_Renderer *render_priv, GlyphInfo *info,
+                 int32_t *leftmost_x,
                  ASS_Vector *pos, ASS_Vector *pos_o,
                  ASS_DVector *offset, bool first, int flags)
 {
+    *leftmost_x = INT32_MAX;
```

Comment:
This function should update instead of overwriting minimum value too. In that case `&current_info->leftmost_x` can be used directly as a parameter.